src/OrbitGl/ThreadStateBar.cpp

```diff
@@ -161,17 +162,30 @@ std::string ThreadStateBar::GetThreadStateSliceTooltip(PrimitiveAssembler& primi
       static_cast<const ThreadStateSliceInfo*>(user_data->custom_data_);
   uint64_t begin_ns = thread_state_slice->begin_timestamp_ns();
   uint64_t end_ns = thread_state_slice->end_timestamp_ns();
-
-  return absl::StrFormat(
+  std::string thread_name = capture_data_->GetThreadName(thread_state_slice->woker_tid());
+  std::string process_name = capture_data_->GetThreadName(thread_state_slice->woker_pid());
```

Comment:
Why are these here and not in the `if`-branch? I think they should be moved into the if